leshan-lwm2m-server-redis/src/main/java/org/eclipse/leshan/server/redis/RedisSecurityStore.java

```diff
@@ -87,9 +90,21 @@ public SecurityInfo getByIdentity(String identity) {
     }
 
     @Override
-    public SecurityInfo getByOscoreIdentity(OscoreIdentity pskIdentity) {
-        // TODO OSCORE to be implemented
-        return null;
+    public SecurityInfo getByOscoreIdentity(OscoreIdentity identity) {
+        try (Jedis j = pool.getResource()) {
+            String recipientHex = Hex.encodeHexString(identity.getRecipientId());
+            String ep = j.hget(endpointByOscoreRecipientIdKey, recipientHex);
```

Comment:
hget get takes byte[] instead of String. 
So I askmyself if this is better to use HexString or byte[] directly ?